.pipelines/cni/pipeline.yaml

```diff
@@ -162,6 +162,15 @@ stages:
       nodeCount: ${NODE_COUNT_CILIUM}
       vmSize: ${VM_SIZE_CILIUM}
 
+  - template: cilium/cilium-overlay-load-test-template.yaml
+    parameters:
+      name: cilium_overlay_hubble_enabled
+      clusterType: overlay-byocni-nokubeproxy-up
+      clusterName: "cilium-over-hubble"

```

Comment:
We need to restrict this to be under 15 characters. AKS creates its own RG that uses the clusterName twice. 

Recommend `cil-over-hub` or `cil-hubble`

Reply:
You need to add a matching delete job for this E2E as well. Make sure to match the `clusterName` for both